### PR TITLE
small edits to centos chapter

### PR DESCRIPTION
**Pull request by Loquacity**
Added SCC channels + general edits.

### CODE REVIEW

File: modules/client-configuration/pages/clients-centos.adoc [hunk 1]
```diff
@@ -41,12 +36,40 @@ systemctl restart taskomatic
 
 == Channel and Repository Management
 
+Before you register {centos} clients to your {productname} Server, check that you have the corresponding {centos} product enabled, and the required channels are fully synchronized.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-centos.adoc:39]: Can we use a less heavy word than "corresponding"? :  right, needed, required, matching --you'll know what I mean ;)

File: modules/client-configuration/pages/clients-centos.adoc [hunk 2]
```diff
@@ -98,14 +114,18 @@ centos8-uyuni-client centos8-appstream
...
 
-.Procedure: Synchronizing {centos} repositories
-
-. In the {productname} {webui}, navigate to menu:Software[Manage], and check every {centos} channel.
-. In the [guimenu]``Repositories`` tab, navigate to the [guimenu]``Sync`` subtab, and click btn:[Sync Now].
-You can also create a regular synchronization schedule on this page.
+* In the {productname} {webui}, navigate to menu:Admin[Setup Wizard] and select the [guimenu]``SUSE Products`` tab.
++
+This dialog displays a completion bar for each product when they are being synchronized.
+* Check the synchronization log file at the command prompt with [command]``tail -f /var/log/rhn/reposync/channel-label.log`` file.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-centos.adoc:125]:
```suggestion
* Check the synchronization log file at the command prompt:
+
----
tail -f /var/log/rhn/reposync/channel-label.log
----
```
And do not forget to explain that "channel-label" is a replaceable.

Comment (Loquacity) [reply modules/client-configuration/pages/clients-centos.adoc]: updated in all of them.

File: modules/client-configuration/pages/clients-centos.adoc [hunk 3]
```diff
@@ -98,14 +114,28 @@ centos8-uyuni-client centos8-appstream
...
+This dialog displays a completion bar for each product when they are being synchronized.
+endif::[]
+ifeval::[{uyuni-content} == true]
+* In the {productname} {webui}, navigate to menu:Software[Manage > Channels], then click the channel associated to the repository.
+Navigate to the [guimenu]``Repositories`` tab, then click [guimenu]``Sync`` and check [systemitem]``Sync Status``.
+endif::[]
+* Check the synchronization log file at the command prompt:
++
+----
+[command]``tail -f /var/log/rhn/reposync/<channel-label>.log``.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-centos.adoc:134]:
```suggestion
tail -f /var/log/rhn/reposync/<channel-label>.log
```

File: modules/client-configuration/pages/clients-debian.adoc [hunk 4]
```diff
@@ -83,9 +83,21 @@ Synchronization can sometimes take several hours.
...
+This dialog displays a completion bar for each product when they are being synchronized.
+endif::[]
+ifeval::[{uyuni-content} == true]
+* In the {productname} {webui}, navigate to menu:Software[Manage > Channels], then click the channel associated to the repository.
+Navigate to the [guimenu]``Repositories`` tab, then click [guimenu]``Sync`` and check [systemitem]``Sync Status``.
+endif::[]
+* Check the synchronization log file at the command prompt:
++
+----
+[command]``tail -f /var/log/rhn/reposync/<channel-label>.log``.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-debian.adoc:98]:
```suggestion
tail -f /var/log/rhn/reposync/<channel-label>.log
```

File: modules/client-configuration/pages/clients-rh.adoc [hunk 5]
```diff
@@ -201,6 +201,27 @@ Ensure you use the correct {rhela} version:
...
+This dialog displays a completion bar for each product when they are being synchronized.
+endif::[]
+ifeval::[{uyuni-content} == true]
+* In the {productname} {webui}, navigate to menu:Software[Manage > Channels], then click the channel associated to the repository.
+Navigate to the [guimenu]``Repositories`` tab, then click [guimenu]``Sync`` and check [systemitem]``Sync Status``.
+endif::[]
+* Check the synchronization log file at the command prompt:
++
+----
+[command]``tail -f /var/log/rhn/reposync/<channel-label>.log``.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-rh.adoc:218]:
```suggestion
tail -f /var/log/rhn/reposync/<channel-label>.log
```

File: modules/client-configuration/pages/clients-sleses.adoc [hunk 6]
```diff
@@ -105,20 +105,28 @@ From the command prompt, add [systemitem]``rhel-x86_64-server-8`` and [systemite
...
-* Check the synchronization log file at the command prompt with [command]``tail -f /var/log/rhn/reposync/channel-label.log`` file.
+endif::[]
+ifeval::[{uyuni-content} == true]
+* In the {productname} {webui}, navigate to menu:Software[Manage > Channels], then click the channel associated to the repository.
+Navigate to the [guimenu]``Repositories`` tab, then click [guimenu]``Sync`` and check [systemitem]``Sync Status``.
+endif::[]
+* Check the synchronization log file at the command prompt:
++
+----
+[command]``tail -f /var/log/rhn/reposync/<channel-label>.log``.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-sleses.adoc:122]:
```suggestion
tail -f /var/log/rhn/reposync/<channel-label>.log
```

File: modules/client-configuration/pages/clients-ubuntu.adoc [hunk 7]
```diff
@@ -247,19 +247,26 @@ You can also create a synchronization schedule from this screen.
...
 ifeval::[{uyuni-content} == true]
-* In the {productname} {webui}, navigate to menu:Software[Manage > Channels], then click on the channel associated to the repository. Navigate to the [guimenu]``Repositories``, then [guimenu]``Sync`` and check [systemitem]``Sync Status``
+* In the {productname} {webui}, navigate to menu:Software[Manage > Channels], then click the channel associated to the repository.
+Navigate to the [guimenu]``Repositories`` tab, then click [guimenu]``Sync`` and check [systemitem]``Sync Status``.
 endif::[]
-* Check the synchronization log file at the command prompt with [command]``tail -f /var/log/rhn/reposync/channel-label.log``.
+* Check the synchronization log file at the command prompt:
++
+----
+[command]``tail -f /var/log/rhn/reposync/<channel-label>.log``.
```
Comment (keichwa) [inline modules/client-configuration/pages/clients-ubuntu.adoc:264]:
```suggestion
tail -f /var/log/rhn/reposync/<channel-label>.log
```

File: modules/client-configuration/pages/clients-centos.adoc (hunk 3 above)
Comment (Loquacity) [reply modules/client-configuration/pages/clients-centos.adoc:134]: Oof, good spot!

**Pull request merged**